bika/lims/utils/analysisrequest.py

```diff
@@ -278,7 +279,7 @@ def _resolve_items_to_service_uids(items):
             continue
 
         # Maybe object UID.
-        portal = portal if portal else api.portal.get()
+        portal = portal if portal else plone_api.portal.get()
```

Comment:
why are you using `plone.api` here? You can also use `api.get_portal()` (which is basically calling plone.api as well, but at least under our control...)

Reply:
Yep, you're right!